pyrs/peaks/peak_collection.py

```diff
@@ -5,7 +5,128 @@
 from typing import Tuple, Union
 from uncertainties import unumpy
 
-__all__ = ['PeakCollection']
+__all__ = ['PeakCollection', 'PeakCollectionLite']
+
+
+def get_strain_conversion_factor(units: str = 'strain') -> float:
+    '''
+    get factor to convert strain to correct units
+
+    Parameters
+    ----------
+    units: str
+        Can be ``strain`` or ``microstrain``
+
+    Returns
+    -------
+    float'''
+    # prepare to return the requested units
+    conversion_factor = 1.
+    if units == 'strain':
+        pass  # data is calculated as strain
+    elif units == 'microstrain':
+        conversion_factor = 1.e6
+    else:
+        raise ValueError('Cannot return units of "{}". Must be "strain" or "microstrain"'.format(units))
+    return conversion_factor
+
+
+def _create_d_reference_array(values: Union[float, np.ndarray],
+                              errors: Union[float, np.ndarray], size: int) -> unumpy.uarray:
```

Comment:
can you add a description of `size`? I'm assuming it's there to deal with the scenario where the user is interested in a subset of the sample points contained in the `PeakCollection` instance, but I am not completely sure of this assumption. A description of this parameter would help clarify.